app/features/ai_resistant_assignment_generator/core.py

```diff
@@ -0,0 +1,28 @@
+from services.logger import setup_logger
```

Comment:
All imports should use absolute paths instead of relative ones, as relative paths won't work in production. Please change all import paths to start with `app`.